.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
I thought this was only necessary if you were using/copying in the local context and using it in the `docker build`? It looks to me like the docker container pulls no files from the local context and it pulls everything it needs from `git clone`, but maybe I'm misunderstanding how the Dockerfile works...?